app/views/ingredients/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= render 'shared/notice' %>
+<div class="container mt-4">
+  <div class="border p-4">
+    <div class="d-flex justify-content-between align-items-center mb-4">
+      <h2>Ingredients</h2>
+      <%= link_to "New ingredient", new_ingredient_path, class: "btn btn-primary btn-sm" %>
+    </div>
+    <table class="table table-striped">
```

Comment:
Show this table (headers) only if ingredients are present, else show message ingredients are not present.
Follow this on every listing page.

Reply:
Done